simulate-event.js

```diff
@@ -281,6 +304,20 @@ module.exports = function (element, type, options) {
 
   event = extend(document.createEvent(eventType), options);
 
+  // Handle differences between `initKeyboardEvent` and `initKeyEvent`.
+  if (initEvent === 'initKeyboardEvent') {
+    if (event[initEvent] === void 0) {
+      initEvent = 'initKeyEvent';
+    } else if (!('modifiersList' in options)) {
```

Comment:
Right, but do we want it to be added when using `initKeyEvent` (because `initKeyboardEvent` is `undefined`) or it doesn't matter?